### DISCUSSION TITLE
add status bar to show url of links without the need to click each one

### DISCUSSION THREAD

**Discussion by brendanmeyer** [zoidsh/meru#269] [Feature Requests] [answered]
There was an issue previously created regarding this (#187)

But as they mention, a useful way to avoid phishing is to be able to see the URL that a link is actually going to.

It is great that there is a confirm link box that appears. (I'm not suggesting to remove this at all, it is extra protection)
But to have to click a link and then have deal with an extra dialog box, it can be time-consuming.
It would be nice to have a status bar (like Chrome) that has the link that it would go to.

Thank you
Brendan

**Comment by timche** [comment]
I've actually planned to add this feature. I'll keep you updated when I found time.

**Comment by timche** [comment, marked as answer]
Added in [v3.0.0-alpha.20](https://github.com/timche/gmail-desktop/releases/tag/v3.0.0-alpha.20).

**Reply by brendanmeyer** [reply]
Great Thank you :)
